docker/worldping-api/worldping-api.ini

```diff
@@ -20,7 +20,7 @@ cert_key =
 admin_key = changeme
 
 [raintank]
-graphite_url = http://graphite-api:8888/
+graphite_url = http://metrictank:6063/
```

Comment:
ok but so it expects a graphite server and we're giving it the metrictank http port, which doesn't really have a graphite http api. that's what confuses me.